caliban/cli.py

```diff
@@ -209,6 +209,15 @@ def image_tag_arg(parser):
       "Caliban will skip the build and push steps and use this image tag.")
 
 
+def dlvm_arg(parser):
+  dlvm_types = [key for key in sorted(conf._dlvm_config('CPU').keys())]
+  parser.add_argument(
+      "--dlvm",
+      help="DLVM base image type. Must be one of  "
+           "{}".format(dlvm_types) + ". If supplied, "
+      "Caliban will skip the build and push steps and use this image tag.")
```

Comment:
hey, I'm going to add comments as I read, so this may be clear later! Naively I would have assumed that the DLVM would be a base image, and that you could still install dependencies on top, right? If that is true, can we call this argument `--base_image`?

Right now, `--image_id` removes any requirement.txt installation; it's truly a flag to skip any build, including even getting your code into the image.

Really, the syntax should be `caliban run e2a4af785bdb`...